api/db-cdr/changelog/db.changelog-2.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="brianfreeman" id="changelog-2" runAlways="true">
+
+    <createTable tableName="criteria">
```

Comment:
Should you add an index on type and parent ID?

Are there any uniqueness constraints here?

Reply:
type and parent id are non unique. We can add indexes here... but should we wait till we have more types and if the table changes because of the types?

Reply:
OK. What about "code"? Is that unique? concept_id?

Reply:
(WRT indexes: I would go ahead and add whatever we're using now, so we get good performance. We can always drop / reset our schema later if needed.)

Reply:
code is non-unique, not sure about concept_id... will have to investigate. Sure i can add indexes